server/src/main/java/org/apache/cloudstack/vm/UnmanagedVMsManagerImpl.java

```diff
@@ -835,9 +823,11 @@ private UserVm migrateImportedVM(HostVO sourceHost, VirtualMachineTemplate templ
             if (CollectionUtils.isNotEmpty(storagePools)) {
                 for (StoragePool pool : storagePools) {
                     if (diskProfileStoragePool.second().getId() != pool.getId() &&
-                            storagePoolSupportsDiskOffering(pool, dOffering) &&
-                            (!profile.getType().equals(Volume.Type.ROOT) ||
-                                    profile.getType().equals(Volume.Type.ROOT) && storagePoolSupportsServiceOffering(pool, serviceOffering))) {
+                            storagePoolSupportsDiskOffering(pool, dOffering)
+                            //FR123 get confirmation from Abhishek
+                            //(!profile.getType().equals(Volume.Type.ROOT) ||
+                                    //profile.getType().equals(Volume.Type.ROOT) && storagePoolSupportsServiceOffering(pool, serviceOffering))
+                            ) {
```

Comment:
As git will keep the history, we can remove the code and move the comment from code to commit message. If it is really needed in code, we can turn it into a javadoc.